pkg/bundle/bundle.go

```diff
@@ -106,7 +103,7 @@ func (b *bundle) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result,
 }
 
 func (b *bundle) reconcileBundle(ctx context.Context, req ctrl.Request) (result ctrl.Result, statusPatch *trustapi.BundleStatus, returnedErr error) {
-	log := b.Log.WithValues("bundle", req.NamespacedName.Name)
+	log := logf.FromContext(ctx)
```

Comment:
suggestion: Previously we were setting the bundle name here (`b.Log.WithValues("bundle", req.NamespacedName.Name)`). Is it intentional to remove that? I think it's useful context

Reply:
Yeah, it was probably a mistake to remove it. My idea was that it is covered by the context set here:

https://github.com/erikgb/trust-manager/blob/8779ee435fe0303e265abadb9417ed520ee8cd72/pkg/bundle/bundle.go#L282-L283

But the following log-line looses context on this.

Reply:
Retained the old behavior now.